src/lib/plugins/log.js

```diff
@@ -23,13 +22,21 @@ module.exports.server = function (serv, settings) {
   serv.on('banned', (banner, bannedUsername, reason) =>
     serv.info(banner.username + ' banned ' + bannedUsername + (reason ? ' (' + reason + ')' : '')))
 
-  serv.on('seed', (seed) => serv.log('seed: ' + seed))
+  serv.on('seed', (seed) => serv.log('Seed: ' + seed))
 
   const logFile = path.join('logs', timeStarted + '.log')
 
   serv.log = message => {
     readline.cursorTo(process.stdout, 0)
-    message = moment().format('MMMM Do YYYY, HH:mm:ss') + ' ' + message
+    const date = new Date()
```

Comment:
why did you replace moment by this fragile code ?

Reply:
moment = heavy, slow